models/ce_music_composition.py

```diff
@@ -0,0 +1,17 @@
+"""Trompa AudioObject model
+"""
+
+from dataclasses import dataclass
+from . import CE_BaseModel, MusicRecording
+
+@dataclass
+class CE_MusicComposition(CE_BaseModel, MusicRecording):
```

Comment:
double-check this, I don't think that MusicComposition is a subclass of MusicRecording: https://schema.org/MusicComposition

Reply:
Changed for CreativeWork, is it ok?

Reply:
It would be better if we created a class "MusicComposition" in sdo_music_composition.py (sdo = schema.org), make it inherit from CreativeWork, and add the new fields to it.

However, I note that we don't use any of the specific MusicCompostion fields that are listed at https://schema.org/MusicComposition, so for now I think that this is fine. I don't think that it's worth adding a lot more code (e.g. all of the schema.org fields) if we don't use any of them.

I see that your `get_mw_audio_1track` method returns a tuple of `(tracks, works, persons)`. One improvement that you could do here is use the field `MusicComposition.composer` to store the list of composers instead of returning it as a separate field, however this is a future change that we can add later.